app/javascript/components/lease_ups/LeaseUpApplicationsTable.js

```diff
@@ -7,33 +7,33 @@ import ReactTable from 'react-table'
 
 import CheckboxCell from 'components/lease_ups/application_page/CheckboxCell'
 import PreferenceRankCell from 'components/lease_ups/application_page/PreferenceRankCell'
+import StatusCell from 'components/lease_ups/application_page/StatusCell'
 import appPaths from 'utils/appPaths'
 import { MAX_SERVER_LIMIT } from 'utils/EagerPagination'
 import { cellFormat } from 'utils/reactTableUtils'
 import { getLeaseUpStatusClass } from 'utils/statusUtils'
 
-import StatusDropdown from '../molecules/StatusDropdown'
-import StatusHistoryPopover from '../organisms/StatusHistoryPopover'
+const CELL_PADDING_PX = 16
 
-const LeaseUpStatusCell = ({ cell, onChange }) => {
-  const applicationId = cell.original.application_id
-  const applicationPreferenceId = cell.original.application_preference_id
+const getCellWidth = (baseSizePx, isAtStartOrEnd = false) => {
+  const padding = isAtStartOrEnd ? 1.5 * CELL_PADDING_PX : CELL_PADDING_PX
```

Comment:
can you maybe add an explanatory comment here? I'm not clear on why it's sometimes 1.5 x 16px